scripts/update_dependencies.sh

```diff
@@ -19,7 +19,7 @@ SOL_PATH="ThePhD/sol2"
 SOL_TAG=v2.17.5
 
 RAPIDJSON_PATH="Tencent/rapidjson"
-RAPIDJSON_TAG=v1.1.0
+RAPIDJSON_TAG=f9d53419e912910fd8fa57d5705fa41425428c35
```

Comment:
This change is extracted to separate PR https://github.com/Project-OSRM/osrm-backend/pull/6906